src/test/java/seedu/address/model/person/PredicateContainerTest.java

```diff
@@ -16,51 +17,12 @@
-        PhoneContainsKeywordsPredicate phonePredicate1 =
-                new PhoneContainsKeywordsPredicate(Arrays.asList("12345678", "87654321"));
-        EmailContainsKeywordsPredicate emailPredicate1 =
-                new EmailContainsKeywordsPredicate(Arrays.asList("alice@gmail.com", "bob@gmail.com"));
-        DepartmentContainsKeywordsPredicate departmentPredicate1 =
-                new DepartmentContainsKeywordsPredicate(Arrays.asList("IT", "HR"));
-        RoleContainsKeywordsPredicate rolePredicate1 =
-                new RoleContainsKeywordsPredicate(Arrays.asList("Manager", "President"));
-
-        NameContainsKeywordsPredicate namePredicate2 =
-                new NameContainsKeywordsPredicate(Collections.singletonList("alice"));
-        PhoneContainsKeywordsPredicate phonePredicate2 =
-                new PhoneContainsKeywordsPredicate(Collections.singletonList("12345678"));
-        EmailContainsKeywordsPredicate emailPredicate2 =
-                new EmailContainsKeywordsPredicate(Collections.singletonList("alice@gmail.com"));
-        DepartmentContainsKeywordsPredicate departmentPredicate2 =
-                new DepartmentContainsKeywordsPredicate(Collections.singletonList("IT"));
-        RoleContainsKeywordsPredicate rolePredicate2 =
-                new RoleContainsKeywordsPredicate(Collections.singletonList("Manager"));
-
-        PredicateContainer predicateContainer1 = new PredicateContainer()
-                .addNameContainsKeywordsPredicate(namePredicate1)
-                .addPhoneContainsKeywordsPredicate(phonePredicate1)
-                .addEmailContainsKeywordsPredicate(emailPredicate1)
-                .addDepartmentContainsKeywordsPredicate(departmentPredicate1)
-                .addRoleContainsKeywordsPredicate(rolePredicate1);
-        PredicateContainer predicateContainer2 = new PredicateContainer()
-                .addNameContainsKeywordsPredicate(namePredicate2)
-                .addPhoneContainsKeywordsPredicate(phonePredicate2)
-                .addEmailContainsKeywordsPredicate(emailPredicate2)
-                .addDepartmentContainsKeywordsPredicate(departmentPredicate2)
-                .addRoleContainsKeywordsPredicate(rolePredicate2);
-
         // same object -> true
+        PredicateContainer predicateContainer1 = getPredicateContainer1();
         assertTrue(predicateContainer1.equals(predicateContainer1));
 
         // same values -> returns true
-        PredicateContainer predicateContainer3 = new PredicateContainer()
-                .addNameContainsKeywordsPredicate(namePredicate1)
-                .addPhoneContainsKeywordsPredicate(phonePredicate1)
-                .addEmailContainsKeywordsPredicate(emailPredicate1)
-                .addDepartmentContainsKeywordsPredicate(departmentPredicate1)
-                .addRoleContainsKeywordsPredicate(rolePredicate1);
+        PredicateContainer predicateContainer3 = getPredicateContainer1();
```

Comment:
This test might be the same as the one above where you are testing for the same object

Reply:
The other tests for the same instance (predicateContainer1 against predicateContainer1), whereas this tests for different instances (predicateContainer1 against predicateContainer3), where both predicateContainer contains the same predicates.